tests/e2e/performance/csi_tests/test_pvc_bulk_creation_deletion_performance.py

```diff
@@ -65,126 +66,119 @@ def init_full_results(self, full_results):
         for key in self.environment:
             full_results.add_key(key, self.environment[key])
         full_results.add_key("index", full_results.new_index)
+        full_results.add_key("interface", Interface_Types[self.interface])
+        full_results.add_key("pvc_size", self.pvc_size)
         return full_results
 
-    """
-    Test to verify PVC creation and deletion performance
-    """
+    def pvc_bulk_create_and_wait_for_bound(self, bulk_size):
+        """
+        Creating a bulk of PVCs and wait until all of the are bounded
```

Comment:
spelling fix: "all of them " instead of "all of the"

Reply:
done